lib/BitwardenJSONSchema.ts

```diff
@@ -1,24 +1,24 @@
 import { z } from './deps.ts';
 
 const URISchema = z.object({
-    match: z.string().nullable(),
+    match: z.union([z.string(), z.number()]).nullable(),
     uri: z.string().nullable(),
 });
 
 const BitWardenItem = z.object({
     name: z.string(),
-    notes: z.string().optional(),
+    notes: z.string().nullable().optional(),
     login: z.object({
         username: z.string().nullish(),
         password: z.string().nullish(),
-        uris: z.array(URISchema),
-    }),
+        uris: z.array(URISchema).optional(),
+    }).optional(),
 });
 
 export const BitWardenJSONExport = z
     .object({
         encrypted: z.boolean(),
-        folders: z.array(z.string()),
+        folders: z.array(z.any()),
```

Comment:
Should this have a schema shape instead of `z.any()`, now that we have an example of what it could be?

Reply:
Yes, that should be safest to detect if they change the schema again.